web/src/hooks/useLatest.ts

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I think we can add `ahooks` to this library directly, without having to implement these hooks separately, what do you think? @guoqqqi @Baoyuantop @SkyeYoung

Reply:
Yes, I think so. Just use the `ahooks` tested and used by many people. @guoqqqi @Silence-dream

Reply:
> 
So I remove custom hooks and add ahooks?

Reply:
> Yes, I think so. Just use the `ahooks` tested and used by many people. @guoqqqi @Silence-dream



> Yes, I think so. Just use the `ahooks` tested and used by many people. @guoqqqi @Silence-dream

So I remove custom hooks and add ahooks?

Reply:
> Yes, I think so. Just use the `ahooks` tested and used by many people. @guoqqqi @Silence-dream

So I remove custom hooks and add ahooks?

Reply:
> > Yes, I think so. Just use the `ahooks` tested and used by many people. @guoqqqi @Silence-dream
> 
> So I remove custom hooks and add ahooks?

Yes.